server/command_hooks.go

```diff
@@ -107,6 +120,8 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 		return p.executeCommandEphemeralOverride(args), nil
 	case commandTriggerDialog:
 		return p.executeCommandDialog(args), nil
+	case commandTestCallback:
```

Comment:
I propose that we change the demo plugin to use the helpers exclusively -- while it's interesting to demo doing both, I think it's a more powerful change to illustrate the feature being used holistically to handle all these callbacks at once.

Reply:
Done :)